pegasus/sites.v3/code.org/public/teach.haml

```diff
@@ -117,7 +117,7 @@ theme: responsive_full_width
         %a.link-button{href: CDO.studio_url("/catalog?grade=grade_10&grade=grade_11&grade=grade_12&grade=grade_9")}
           =hoc_s(:grade_level_label_high)
 
-%section.resources.bg-neutral-light.centered
+%section.resources.bg-neutral-light
```

Comment:
Checking my understanding here: this is *not* centered because it doesn't have an h2 or p.desc tab?

Reply:
Ah but the parts of the partial with the p.desc tag *will be* centered?

Reply:
^^ I answered my own questions ignore me

Reply:
That `.centered` class can be applied to any element, so having it on the `section` was applying it too broadly and caused this issue on tablet/mobile 🥲